### DISCUSSION TITLE
Simplify Favorites/Recent. Add ROM completed status.

### DISCUSSION THREAD

**Discussion by gitleecher** [OnionUI/Onion#1506] [Ideas]
It would be nice if we could track which ROMs we have competed. Added via the ROM context menu. And showing a checkmark where the favorite star is.

To view completed ROMs; having a way to cycle in the Favorites UI screen between Favorites / Recent / Completed would make theming easier if we could drop the recent tab from the Main UI.

-Dean

**Comment by oherau** [comment]
Maybe all info regarding game completion and trophies could be centralized in one single app
[https://github.com/OnionUI/Onion/discussions/1465](https://github.com/OnionUI/Onion/discussions/1465)
... and displayed as well in game detail
[https://github.com/OnionUI/Onion/discussions/1528](https://github.com/OnionUI/Onion/discussions/1528)

what do you think ?
